x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/three_way_diff/rule_upgrade/translations.tsx

```diff
@@ -89,37 +89,73 @@ export const RULE_HAS_CONFLICTS = (count: number) =>
     {
       values: { count },
       defaultMessage:
-        '{count} {count, plural, one {field has a conflict} other {fields have conflicts}}. Please review and provide a final update.',
+        '{count} {count, plural, one {field has a conflict} other {fields have conflicts}}. Review and provide a final update.',
     }
   );
 
 export const RULE_HAS_SOFT_CONFLICTS_DESCRIPTION = i18n.translate(
   'xpack.securitySolution.detectionEngine.rules.upgradeRules.fieldUpgradeState.ruleHasSoftConflictsDescription',
   {
     defaultMessage:
-      'Please review and accept conflicts. You can also keep the current version without the updates, or accept the Elastic update but lose your modifications.',
+      'We auto-resolved the conflicts between your changes and the Elastic update. Review the resolved conflicts and do one of the following:',
```

Comment:
Slightly shorter option. 

```suggestion
      'We auto-resolved the conflicts between your changes and the Elastic update. Review them and do one of the following:',
```